clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -95,14 +98,14 @@ func (ste *simpleExecutor) ExecuteTest(ctx Context, conf *api.Config) *errors.Er
 	return errList
 }
 
-// ExecuteTestSteps executes all test steps provided in configuration
-func (ste *simpleExecutor) ExecuteTestSteps(ctx Context, conf *api.Config) *errors.ErrorList {
+// prepareTestNamespaces prepares k8s namespaces for the test.
+func (ste *simpleExecutor) prepareTestNamespaces(ctx Context, conf *api.Config) error {
 	automanagedNamespacesList, staleNamespaces, err := ctx.GetClusterFramework().ListAutomanagedNamespaces()
 	if err != nil {
-		return errors.NewErrorList(fmt.Errorf("automanaged namespaces listing failed: %v", err))
+		return fmt.Errorf("automanaged namespaces listing failed: %v", err)
```

Comment:
Nit: s/%v/%w.

Reply:
seems like a fixit thing to change is in many different places :)

Reply:
Good catch, done!